lib/identity_cache/query_api.rb

```diff
@@ -80,7 +80,13 @@ def fetch_multi(*ids)
       def record_from_coder(coder) #:nodoc:
         if coder.present? && coder.has_key?(:class)
           record = coder[:class].allocate
-          unless coder[:class].serialized_attributes.empty?
+          empty_serialized_attrs =
+            if defined?(ActiveRecord::Type::Serialized)
+              coder[:class].columns.find { |t| t.cast_type.is_a?(ActiveRecord::Type::Serialized) }.nil?
+            else
+              coder[:class].serialized_attributes.empty?
+            end
+          unless empty_serialized_attrs
```

Comment:
Nope, because I didn't add it.  https://github.com/Shopify/identity_cache/pull/88